webapp/channels/src/components/suggestion/suggestion_list.tsx

```diff
@@ -46,19 +43,13 @@ export default class SuggestionList extends React.PureComponent<Props> {
     };
     contentRef: React.RefObject<HTMLUListElement>;
     wrapperRef: React.RefObject<HTMLDivElement>;
-    itemRefs: Map<string, any>;
```

Comment:
How come we took the refs out? Isn't that a better way to locate the element you need in `scrollToItem`?